### ISSUE TITLE
Why there are fewer annotations for the data set？

### ISSUE THREAD

**Issue by KKDDD** [bdd100k/bdd100k#64]
When I used the script provided by the warehouse to convert the BDD format to COCO format, for the training set, I found that the number of images in the COCO format was only 69863 instead of 70,000 in BDD. The number of images in the validation set remained the same as before.

**Comment by XiaLiPKU** [comment]
Currently, for detection conversion, we choose remove those images without any labels.
You may see this line: https://github.com/bdd100k/bdd100k/blob/7fbb4a902f2076938bede3016ad83b3dbabd6901/bdd100k/label/to_coco.py#L243.

We may add it as an option later.

**Comment by XiaLiPKU** [comment]
I just checked it, and feel sorry to say, these labels are missed accidentally.
Still, thanks for your finding

**Comment by KKDDD** [comment]
> I just checked it, and feel sorry to say, these labels are missed accidentally.
> Still, thanks for your finding

ok,I want to know whether the 20000 test images of 2D target detection provide real labels, I did not find it on the Internet

**Comment by XiaLiPKU** [comment]
We prepared a testing server: https://competitions.codalab.org/competitions/30624.
You can evaluate your algorithm's performance upon testing set there.

Given this server, we will not publish the labels for the testing set.
